pkg/sql/opt/optbuilder/insert.go

```diff
@@ -1054,9 +1057,9 @@ func (mb *mutationBuilder) mapColumnNamesToOrdinals(names tree.NameList) util.Fa
 	var ords util.FastIntSet
```

Comment:
Can you add to the `mapColumnNamesToOrdinals` to make it clear that it skips mutation columns?